homeassistant/components/google_assistant/smart_home.py

```diff
@@ -71,6 +71,24 @@ async def _process(hass, data, message):
     return {"requestId": data.request_id, "payload": result}
 
 
+async def async_devices_serialize(hass, config, agent_user_id):
```

Comment:
Can you call this `async_devices_sync_response` ? We have a similar name/method already at the bottom of this file.

Reply:
Sure make sense.

Reply:
Actually. with the change to add create_sync_response, the original name make more sense since it will only serialize devices.